lib/fika/cli.ex

```diff
@@ -21,12 +35,26 @@ defmodule Fika.Cli do
     {opts, rest} = OptionParser.parse!(rest, options)
 
     main_file = List.first(rest) || "main.fi"
-    function = opts[:function] || "main.start()"
+    function = opts[:function] || "start()"
+
+    unless File.exists?(main_file) do
+      IO.puts("File #{main_file} not found.")
+      System.halt(1)
+    end
 
     {:module, module} = Fika.Code.load_file(main_file)
-    Logger.debug("Calling :#{module}.#{function}")
-    {result, _binding} = Code.eval_string(":\"#{module}\".#{function}")
-    IO.inspect(result)
+
+    fn_not_found_msg = "Function #{function} not found."
+
+    try do
+      Logger.debug("Calling :#{module}.#{function}")
+      {result, _binding} = Code.eval_string(~s':"#{module}".#{function}')
+      IO.inspect(result)
+    rescue
+      UndefinedFunctionError ->
+        IO.puts(fn_not_found_msg)
+        System.halt(2)
```

Comment:
Any particular reason why the exit code is 2?

Reply:
Just so different errors return different codes. No problem in changing this to another number if you want to